package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@celonis/content-cli",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
You probably need to bump this once more as PR's have already been merged to master and this won't get published